app.js

```diff
@@ -293,7 +293,8 @@ ipcMain.on('clear-apv', () => {
 });
 
 function createBroadcastFiles(arg) {
-  const userDataPath = electron.app.getPath('userData');
+  const liveFeedLocation = store.get('userPrefs.app.live-feed-location');
+  const userDataPath = (liveFeedLocation === 'default' || !liveFeedLocation) ? electron.app.getPath('desktop') : liveFeedLocation;
```

Comment:
I hope `electron.app.getPath('desktop')` works for windows/linux as well. only tested on mac. 

cc. @navdeepsinghkhalsa

Reply:
It's officially documented, so they should be doing the legwork, here. I appreciated the find. Good work :)